build/root/Makefile

```diff
@@ -578,3 +578,18 @@ else
 bazel-release:
 	bazel build //build/release-tars
 endif
+
+ifeq ($(PRINT_HELP),y)
+define DIND_RELEASE_HELP_INFO
+# Build dind test images with bazel
+#
+# Example:
+# make dind
+endef
+dind:
+  @echo "$$DIND_RELEASE_HELP_INFO"
```

Comment:
should this be a tab rather than 2 spaces?

Reply:
Whoops, vim doesn't let me use tabs :(